packages/portfolio-contract/tools/wallet-offer-tools.ts

```diff
@@ -121,7 +120,7 @@ export const makeWallet = (
         providePurse,
       );
       const refund: AmountKeywordRecord = await deeplyFulfilledObject(
-        objectMap(payouts, async pmt => wallet.deposit(await pmt)),
+        objectMap(payouts, async pmt => E.when(pmt, wallet.deposit)),
```

Comment:
No longer has `await`
```suggestion
        objectMap(payouts, pmt => E.when(pmt, wallet.deposit)),
```

save for the one on 157

Reply:
I'm inclined to take advantage of the fact that ci is green.

here's hoping I find a moment to tidy this up in my next PR